packages/mirinae/src/inputs/dropdown/select-dropdown/PSelectDropdown.vue

```diff
@@ -94,16 +91,23 @@ const state = reactive({
         }
         return res;
     }, {})),
+    innerText: computed<string>(() => {
+        // TODO: need to check at console for translation.
+        if (typeof (props.innerLabel) === 'string') {
+            return state.selectedItems?.length > 0
+                ? `${state.selectedItems[0].label || state.selectedItems[0].name || ''}`
+                : (props.innerLabel || 'Label');
```

Comment:
Suggestion: How about leaving it empty when the innerLabel prop is not present? If there is a placeholder, should we display the placeholder?

Reply:
Since `innerText` is a state that calculates the value of the text, including the default placeholder of the `dropdown button`, it seems appropriate for the placeholder to be included when the value is empty!